kythe/go/platform/tools/kzip/filtercmd/filtercmd.go

```diff
@@ -37,9 +38,10 @@ import (
 type filterCommand struct {
 	cmdutil.Info
 
-	input    string
-	output   string
-	encoding flags.EncodingFlag
+	input     string
+	output    string
+	encoding  flags.EncodingFlag
+	languages flagutil.StringList
```

Comment:
I think you want `flagutil.StringSet`. Obviates the subsequent conversion to `stringset.Set`.

Reply:
I tried but I'm getting these errors:
```
kythe/go/platform/tools/kzip/filtercmd/filtercmd.go:91:34: languages.Empty undefined (type flagutil.StringSet has no field or method Empty)
kythe/go/platform/tools/kzip/filtercmd/filtercmd.go:97:23: languages.Empty undefined (type flagutil.StringSet has no field or method Empty)
kythe/go/platform/tools/kzip/filtercmd/filtercmd.go:99:21: languages.Contains undefined (type flagutil.StringSet has no field or method Contains)
```

And when I'm trying to use StringSet as Set (just to see if that works) I get this:

```
var languages stringset.Set = c.languages
kythe/go/platform/tools/kzip/filtercmd/filtercmd.go:89:32: cannot use c.languages (variable of type flagutil.StringSet) as type stringset.Set in variable declaration
````

Am I missing some basic golang understanding?

Reply:
Right, you'll need to cast it to a `stringset.Set` for example: https://github.com/kythe/kythe/blob/145859480b64849c1911cebfc09c42be69835476/kythe/go/test/tools/empty_corpus_checker/empty_corpus_checker.go#L86

The flag type doesn't have an `Empty` method, but does have `Len`.  It may be reasonable to add `Empty` though.